CS.API/Controllers/TicketsController.cs

```diff
@@ -31,7 +32,7 @@ public async Task<IActionResult> GetAll([FromQuery] TicketFilter filter, Cancell
 
             var role = HttpContext.User.FindFirstValue(ClaimTypes.Role);
 
-            if (role == "User")
+            if (role == UserRoles.User)
```

Comment:
same as in the Security project - it's better not use the string literals, and move then to a constants somewhere